### DISCUSSION TITLE
Support for BLE Commissioning?

### DISCUSSION THREAD

**Discussion by javierbarellano4v** [matter-js/matter.js#183] [General]
I've noticed much discussion on BLE in Discord but I thought I would pose my basic questions here.

Is **BLE-based commissioning** supported or partially supported in matter.js?

I've been away from node-matter/matter.js for some time, prior to migration to matter.js.

I've been trying to demonstrate BLE-based commissioning using the Matter SDK between Amazon Alexa and either the examples apps, lighting-app or chip-tv-app. The SSID and credentials appear to pushed to the app but then the commissioning fails.

So, in any case, I'm hoping that matterl.js will support BLE-commissioning.

I don't have the cycles to dev but I can test its interoperability with some of the ecosystems when I have time.

Thank you.

**Comment by Apollon77** [comment]
We use an own Discussion group for feature requests :-) Please follow #177
